Mathlib/RingTheory/Flat.lean

```diff
@@ -76,6 +91,121 @@ instance self (R : Type u) [CommRing R] : Flat R R :=
       lift.tmul, Submodule.subtype_apply, Algebra.id.smul_eq_mul, lsmul_apply]⟩
 #align module.flat.self Module.Flat.self
 
+variable (M : Type v) [AddCommGroup M] [Module R M]
+
+/-- A reformulation of the flat property. -/
```

Comment:
If you add a docstring to this lemma, please add a bit more detail: rephrasing the flat property as <put details here>. That way it will be more useful to readers of the docstring.

Reply:
As this is simply rephrasing the definition in term of the other possible definition of the canonical map, perhaps no docstring is actually a better solution.